EffectAndStructuredConcurrency.md

```diff
@@ -0,0 +1,202 @@
+# Semantics of Structured Concurrency + Effect<R, A>
```

Comment:
I don't think this doc will show up on the Arrow Website this way.
Do we need to move it to the KDoc of the `Effect` interface? Or do we need to put it in a specific place and include it into the sidebar of the Arrow Core website?

Reply:
It's now in Arrow Core and Fx